deploy/csi-rclone/templates/csi-nodeplugin-rclone.yaml

```diff
@@ -154,6 +156,10 @@ spec:
         {{ toYaml . | nindent 8 }}
     {{- end }}
       volumes:
+      - hostPath:
+          path: /tmp/{{.Release.Namespace}}-{{.Release.Name}}-{{.Release.Revision}}
```

Comment:
suggestion: Can we mount at a path that is not `/tmp/...` because it indicates that the contents can be deleted without consequences, not sure if the os can then clean this stuff up randomly